### ISSUE TITLE
How to query a label with a space in the name?

### ISSUE THREAD

**Issue by jevansaks** [izuzak/github_issue_stats#14]
I'm trying to use the history command against our repo (microsoft/microsoft-ui-xaml). It works fine for the "bug" label but I'm trying to query with the ["feature proposal" label](https://github.com/microsoft/microsoft-ui-xaml/labels/feature%20proposal) and I can't get it to show results. I've tried:

```
--labels "feature proposal"
--labels feature+proposal
--labels feature%20proposal
```

And they all still show 0. Any ideas?

**Comment by izuzak** [comment]
Sorry for the delay in responding to this, @jevansaks 😓 I've been busy battling some health issues 

You're right -- this wasn't possible before but should be possible now with this syntax:

```
--labels "feature proposal"
```

Let me know if you notice any other problems! And thanks for reporting this 💙

**Comment by izuzak** [comment]
Note: I published a new version of the package (0.6.0) which has this fix
